### PR TITLE
Xen net driver refactoring

### PR DESCRIPTION
**Pull request by restonich**
For the sake of codestyle and understanding.

### CODE REVIEW

**Comment by restonich** [comment]
Need your review, @AntonKozlov

**Comment by AntonKozlov** [comment]
Will take some time...
Why mine review exactly? Anyone can participate!

**Comment by restonich** [comment]
We've fixed bugs, so now it initializes correctly. There're still few things like when we write information to XenStore with `xenstore_write`, it writes strings with NUL character, which is not expected behavior.

File: src/drivers/net/xen_net/Mybuild [hunk 1]
```diff
@@ -1,7 +1,10 @@
 package embox.driver.net
 
-module xen_pv_net {
-	source "xen_pv_net.c"
+module xen {
+	source "xen_net.c",
+			"netfront.c",
+			"xs_info.c"
+
```
Comment (Deryugin) [inline src/drivers/net/xen_net/Mybuild]: Remove one blank line, please

**Comment by AntonKozlov** [comment]
Gently remainder: all participants of xen19 course are invited to review on this and all other xen19's PRs

**Pull request merged**